src/styles/theme.js

```diff
@@ -49,6 +51,8 @@ const gradients = {
   info: `linear-gradient(90deg, ${colors.blue} 0%, ${colors.purpleLight} 100%)`,
   success: `linear-gradient(90deg, ${colors.green} 0%, ${colors.blue} 100%)`,
   warning: `linear-gradient(90deg, ${colors.orangeDark} 0%, ${colors.orange} 100%)`,
+  blue: `linear-gradient(180deg, ${colors.fountainBlue} 0%, ${colors.springGreen} 100%)`,
+  blueFromToptoBottom: `linear-gradient(${colors.fountainBlue}, ${colors.springGreen}) padding-box, linear-gradient(to bottom, #48B2B7, #06FC9D) border-box`,
```

Comment:
Maybe the name of this should be turquioseGradient?

Reply:
New naming suggestions: blueGreen instead of blue
The other is it going from green to blue from the top? I am not really familiar with gradients. But in that case greenBlue?